### DISCUSSION TITLE
No audio upload possible?

### DISCUSSION THREAD

**Discussion by TheNomad11** [mawoka-myblock/ClassQuiz#329] [Q&A]
I wanted to create some listening exercises and tried to upload audio files. This does not seem to be possible? I haven't found a way to do so. Or are there some settings I didn't see? (A workaround is to convert mp3 to mp4 files, but this would be combersome in the long run. )

**Comment by mawoka-myblock** [comment]
Created an issue for that.

**Reply by TheNomad11** [reply]
Thanks!
